### PR TITLE
Add Jenkinsfile 

### PR DESCRIPTION
**Pull request by himanshuupadhyay101**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    timeout(time: 30, unit: 'MINUTES')
+  }
+
+  agent {
+    kubernetes {
+      label 'worker-docker'
+      inheritFrom 'default'
+
+      containerTemplates([
+        containerTemplate(name: 'golang', image: 'golang:1.12.5', ttyEnabled: true, command: 'cat')
```
Comment (benwaffle) [inline Jenkinsfile:15]: go 1.12 is very old, let's use the latest go 1.x

```suggestion
        containerTemplate(name: 'golang', image: 'golang:1', ttyEnabled: true, command: 'cat')
```

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+          sh "cd play && ./build-play ${params.SEM_INFO} ${params.VERSION13}"
+          sh "cd play && ./play/build-play-builder ${params.SEM_INFO} ${params.VERSION13}"
+        }
+      }
+    }
+
+    stage('Upgrade node docker image') {
+      steps {
+        container('ruby') {
+          sh "./node/build-node ${params.SEM_INFO} ${params.VERSION12}"
```
Comment (gheine) [inline Jenkinsfile:52]: you'll need to `cd node && ./build`... here too

File: Jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,62 @@
+properties([pipelineTriggers([githubPush()])])
+pipeline {
+  options {
+    disableConcurrentBuilds()
+    buildDiscarder(logRotator(numToKeepStr: '3'))
+    timeout(time: 30, unit: 'MINUTES')
+  }
+  parameters {
+     string(name: 'SEM_INFO', defaultValue: 'sem-info tag latest', description: 'SEM-INFO ')
```
Comment (gheine) [inline Jenkinsfile:9]: remove this, as `sem-info` won't be available. Replace everywhere below with `git describe --tags --dirty`

Comment (gheine) [reply Jenkinsfile:9]: Actually won't need this at all. Simply replace with `VERSION.printable()`

File: Jenkinsfile [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+  }
+
+  environment {
+    ORG = 'flowcommerce'
+    APP_NAME = 'docker'
+    GOPRIVATE='github.com/flowcommerce'
+  }
+
+  stages {
+    stage('Checkout') {
```
[the remark this answers is not in the document]
Comment (himanshuupadhyay101) [reply Jenkinsfile:34]: done

**Pull request merged**